ArcGISRuntimeSDKQt_CppSamples/Analysis/PerformValveIsolationTrace/PerformValveIsolationTrace.cpp

```diff
@@ -216,7 +238,7 @@ void PerformValveIsolationTrace::connectSignals()
 
     if (UtilityElementTraceResult* utilityElementTraceResult = dynamic_cast<UtilityElementTraceResult*>(utilityTraceResultList->at(0)))
     {
-      if (utilityElementTraceResult->elements(this).empty())
+      if (utilityElementTraceResult->elements().empty())
```

Comment:
sorry - I guess my comment here wasn't that clear. I _think_ that the call to `elements` accepting the `this` ptr is returning a set of objects (elements?) with this as their parent. My original comment was that those newly allocated objects are not referenced so are effectively a leak. I think we make the same call further down as well so you would be better to get them once and reuse the result.